docs/kintoblocks.md

```diff
@@ -93,36 +74,6 @@ These are accessed during the build process so when updating them you need to ad
 
 The section on Environment Variables lives on the **KintoBlock Edit** page, towards the end of the form. They are specific to a **branch** of a KintoBlock, so if you switch branches and want to use that particular branch in a KintoApp you will need to input the **Environment Variables** on that branch too.
 
-
-### Points to remember about Environment Variables
-
-- These can only be edited by the creator of a KintoBlock.
-- If you are building a block that is shared, will be included in multiple applications, and will require information from the end user, it is best to use *Custom Paramaters*.
-
-
-## Custom Parameters
-
-Custom Parameters are used to pass information to your **KintoBlock Microservice** . These values are editable by the end user in the case of shared KintoBlocks, and also overridable per environment at a KintoApp level for cases where multiple apps are using the same **KintoBlock**. For more insight into KintoHub's structure please refer to [Thinking Kinto](https://docs.kintohub.com/docs/thinking-kinto). In the case of shared KintoBlocks custom params, custom key/value data can be added to the headers which can give you context as to what and who is calling your block.
-
-When creating KintoBlock you can assign default values to your Custom Parameters and these can be marked required if they are necessary for your KintoBlock to function, for example a Sendgrid API or Secret.
-
-Custom Parameters are accessible inside your code within each call. You must specify when to use them, and whether they are required or optional via API doc headers, in a **Node.js** application adding the header to a call will look like this, values with `[`these brackets`]` are optional:
-
-`@apiHeader (Config) {String} sendgridAPI` - A required value 
-
-`@apiHeader (Config) {String} [sendgridSecret]` - An optional value
-
-
-
-This will be injected as a header into this call, and accessible in the usual way you would access headers. In a **Node.js / Express** KintoBlock it would look like this: 
-
-```
-  const githubToken = req.get('sendgridAPI')
-```
-
-### Points to remember about Custom Parameters 
-- Use these to set variables that the customer will need to set or change, like API keys for external APIS.
-- Set default values to guide the end user.
-- Mark important **Custom Parameters** as required so they are not accidentally left out.
+> Note: Environment Variables can be edited only by the creator of a KintoBlock.
```

Comment:
When you say edited => it feels like i cannot set the values? I should be able to set the values => this needs to be more clear. That adding new or removing existing env vars are only possible by the KB owner. but using deployment configuraiton you can set them